Lib/argparse.py

```diff
@@ -398,13 +398,17 @@ def _format_actions_usage(self, actions, groups):
                 raise ValueError(f'empty group {group}')
 
             try:
-                start = actions.index(group._group_actions[0])
+                start = _sys.maxsize
+                for item in group._group_actions:
+                    index = actions.index(item)
+                    if index < start:
+                        start = index
```

Comment:
I think that you can use `min()`:
```suggestion
                start = min(actions.index(item) for item in group._group_actions)
```
or
```suggestion
                start = min(map(actions.index, group._group_actions))
```